src/tokens.ts

```diff
@@ -269,6 +269,10 @@ export interface IGitExtension extends IDisposable {
    * @param path Path from which the top Git repository needs to be found
    */
   showTopLevel(path: string): Promise<Git.IShowTopLevelResult>;
+
+  taglist: Git.ITag[];
+  tags(path: string): Promise<Git.ITagResult>;
+  tag_checkout(path: string, tag: string): Promise<Git.ICheckoutResult>;
```

Comment:
Add please documentation here.

```suggestion
  tags(path: string): Promise<Git.ITagResult>;
  tag_checkout(path: string, tag: string): Promise<Git.ICheckoutResult>;
```